docs/introduction/start-building-your-application.md

```diff
@@ -48,6 +48,35 @@ When you install new project, locales are set like this
 
 In case you want to change domain locale read [locale settings](./how-to-set-up-domains-and-locales.md#3-locale-settings) or in case you want to change default administration locale read [locale in administration](./how-to-set-up-domains-and-locales.md#36-locale-in-administration).
 
+## Set up domain type
+
+Each domain can be configured with a type. The type can be either `B2C` (Business-to-Consumer) or `B2B` (Business-to-Business). This option is set using the `type` parameter in the domain configuration.
```

Comment:
_protip_

Having individual sentences on separate lines improves diff readability in the future

Reply:
Thx a lot, changed